client/components/gsuite/docs/new-user-list.tsx

```diff
@@ -9,14 +9,15 @@ import {
 	GSuiteNewUserField,
 	newUsers,
 } from 'calypso/lib/gsuite/new-users';
-import type { SiteDomain } from 'calypso/state/sites/domains/types';
 
-const domainOne: SiteDomain = {
+// `GSuiteNewUserList` expects domains to use the `ResponseDomain` type, but we use `any` here to
+// avoid overly verboes declarations.
```

Comment:
I'm fine setting it to `any` since this is an example (doc) but there is a typo in the comment. `Verboes` should be `verbose`. And since you are already editing this file, what about settings the props in alphabetical order? :)

Reply:
Thanks, good catch! Have pushed a fix for the typo. Took the opportunity to sort the props as well.

Reply:
Seems reasonable to rename the type to something a bit more generic as well. I'm not all that familiar with the codebase just yet, but given the "universal" purpose of this type, I'm thinking that `Domain` could be a decent candidate.